src/journal/urls.py

```diff
@@ -24,15 +35,23 @@
     url(r'^article/(?P<article_id>\d+)/galley/(?P<galley_id>\d+)/view/',
         views.view_galley,
         name='article_view_galley'),
-    url(r'^article/(?P<identifier_type>.+?)/(?P<identifier>.+?/.+?)/table/(?P<table_name>.+)$',
+    url(r'^article/(?P<identifier_type>{0})/(?P<identifier>\d+)/table/(?P<table_name>.+)$'
+        ''.format("|".join(IDENTIFIER_TYPES)),
+        views.download_table,
+        name='article_table'),
```

Comment:
Could use a comment explaining why we duplicate this urls with the same name

Reply:
Added.